### PR TITLE
Funny title about yet another QoL patch of mine

### PR DESCRIPTION
**Pull request by ThePainkiller**
## Sorry, out of witty ideas!

- Termites (not the kind that spawn from drills) are no longer dense, meaning they won't friendly fire themselves, and have to be targeted manually to shoot (much like roaches)
- Adds Cyberpunk Bartender outfit to casual dress selection and the clothing machines. Mix lives, change drinks.
Jill Stingray
Gillian
- Crafting Station can now make cheap 12mm barrels, and non-cheap versions but at double the plasteel cost. Also fixes the calibers displayed on it from .35 and .40 to 9mm and 10mm respectively
- Several guns fixed on their gun parts crafting where they were previously uncraftable
- Switches some grips and parts around, namely the Motherfucker takes steel parts, and swaps the Mistral and Minotaur's grip requirement (Mistral takes rubber as it's better, Minotaur takes plastic)
- Buffs the Bison to do a fair bit more damage, making it an actual upgrade worth machining for besides massive drip
- Homemade slide shotgun frame fixed, no longer misrepresented as a "homemade kalash frame"
- Novakovic now takes a steel barrel, Mosin requires a good quality barrel
- Bakelite grip on the Solmarine, as it's a literal dated gun
- Takeshi now has a proper barrel for its caliber
- Duty takes a plastic grip as most Blackshield guns intended by design (that aren't high end)
- Reverts the penetration and one handed penalty debuffs on the gyrogrip as it was too punishing for a basic mod (The vibrocomp is justified as it's a massive 50% reduction)
- "Cherubim" Durand paintjob now looks like a Seraph mech as originally intended
- Deletes duplicate reticence sprites
- Renames the "goatee" facial hair to what it truly is: A "circle beard".
- Adds dali moustache, goatee, Guy Fawkes, Robotnik moustache and moustacheless versions of long beard and full beard to facial hair selection
PRRRRRRROMOTION
_Get yourself a PRRRRRRRRROMOTION today!_
- New Jill haircut (See screenshot above)
- Syndie paintjob for APLUs is now on the correct Eris palette for syndie stuff
image
- Improves M.A.I.D. mech by giving it Marshal colors
- Manual action mechanism renamed to bolt action mechanism
- Curtains are now really made of cloth. No, I mean, really! Not only do they have cloth as a proper material now, they're also crafted with it! (Furniture tab of crafting menu) Except the plastic curtains. Those require plastic, duh!
- Fixes the boxes of carpets from the bioprinter not spawning anything at all
- Cheap spear is now properly a spear, in all senses of the word
- Curtains can be taken down with a wirecutting tool now, reclaiming their materials. CDDA/Zomboid reference?
- New in-hands for cyberpunk bartender outfits
- Knuckledusters actually affect punching strength now!
- Added Cheap spear wielded icons
- Many map changes, mostly QOL: Better fire control on Dorms, actual functional space tiles on the pod room, Vacant Tech shops now uses its own area and is properly powered, atmosed, fire measures, and more anti break in, moved the sink from the kitchen so that the farmbot doesn't get stuck
- Scavenger gloves, made by combining handmade combat gloves with plasteel knuckle dusters and plasteel, bonus to punches and extra armor, colorable to fit the aesthetic of the Scavenger armor
- Ports Makeshift Blade, [Staff, Halberd](https://github.com/discordia-space/CEV-Eris/pull/7873) and [Battle Axe](https://github.com/discordia-space/CEV-Eris/pull/7937) from Eris
- New melee option for the Foreman: The "Render Slayer" Greatsword, CLANG sound not included (but it has a snowflake slash sound)
- Replaces the Foreman's sledge on their office for a weapon selection guncase between the Sledgehammer and the Greatsword
- Made it so that cheap holsters can't sheathe the Cheap Spear (now that it's ACTUALLY a spear!)
- Adds Bulldozer shield back sprite, previously missing
- Oni Club no longer locked to 40 damage when dual wield, now it takes mods into consideration
- Many typo fixes

## Changelog
:cl:
add: Cyberpunk Bartender outfit to casual dress selection, also on Bartender's locker and clothing vendor
add: Jill haircut. Goatee, Guy Fawkes, Dali, Robotnik moustaches and moustacheless versions of long beard and full beard. Renames the current goatee to "circle beard".
add: Craftable curtains to furniture tab
add: Improvised blade, battle axe, staff and halberd from Eris
add: Foreman now gets a case on their office to pick a weapon of their choice: Foreman's Sledgehammer, or the new Render Slayer Greatsword
add: Scavenger gloves, craftable with combat gloves and plasteel knuckle dusters. Good armor, extra punching damage, interchangeable colors to fit both Scavenger Suit variants
tweak: Curtains can be deconstructed using tools of wire cutting quality
tweak: Swapped around some grips on guns that weren't high grade, put rubber on guns that are actually high grade
balance: 12mm steel and normal barrels to ammo crafting station, the good version costs eight plasteel instead of four, making marshals a way cheaper alternative
balance: Bison now has 0.5 more damage and pen
fix: Fixes the boxes of carpets from the bioprinter not spawning anything at all
fix: Fixed a lot of guns being impossible to craft due to their components missmatching
map: Drippier dorms, fixed space shuttle dorms overlay, moved the sink on the Kitchen away so that the farmbot won't brick itself trying to reach it, Vacant Tech shops takes its proper area now, better fireproofing of dorms and tech shops, vent in telecomms starts on so that it doesn't vacuum the area, and more
soundadd: Snowflake slashing sound for the Foreman's sword
imageadd: Better Syndie paintjob and MAID exosuit sprite
imagedel: Deleted duplicated "recitence" (sic) icons
spellcheck: fixed a few typos
code: Knuckledusters now actually add to unarmed damage formula
/:cl:

### CODE REVIEW

File: code/game/machinery/crafting_station.dm [hunk 1]
```diff
@@ -150,19 +150,22 @@
...
+			choice = input(user, "Which type of part do you want to craft?") as null|anything in list("Small arms", "Long arms", /*"Heavy parts"*/"Cheap parts", "Grips")
 			switch(choice)
-				if("Small parts")
+				if("Small arms")
 					choice = input(user, "Which type of part do you want to craft?") as null|anything in list("Barrels", "Mechanisms")
 					switch(choice)
 						if("Barrels")
 							required_resources = materials_sbarrel
-							choice = input(user) as null|anything in list(".35 barrel", ".40 barrel")
+							choice = input(user) as null|anything in list("9mm barrel", "10mm barrel")
```
Comment (Trilbyspaceclone) [inline code/game/machinery/crafting_station.dm:160]:
```suggestion
							choice = input(user) as null|anything in list("9mm barrel", "10mm barrel", "12mm barrel")
```

File: code/game/machinery/crafting_station.dm [hunk 2]
```diff
@@ -243,7 +249,7 @@
 								if("SMG mechanism")
 									required_resources = materials_smechanism_steel
 									items_to_spawn = list("" = /obj/item/part/gun/mechanism/smg/steel)
-								if("Manual-action mechanism")
+								if("Bolt-action mechanism")
```
Comment (Trilbyspaceclone) [inline code/game/machinery/crafting_station.dm:252]: needs change to the 241 line

File: code/game/machinery/crafting_station.dm [hunk 3]
```diff
@@ -256,12 +262,12 @@
 				if("Grips")
 					choice = input(user) as null|anything in list("Plastic grip", "Bakelite grip", "Wooden grip") //"Rubber grip", "Excelsior grip",
 					switch(choice)
-						if("Plastic grip")
```
Comment (Trilbyspaceclone) [inline code/game/machinery/crafting_station.dm:259]: needs chnages to the 263 line

File: code/modules/clothing/gloves/miscellaneous.dm [hunk 4]
```diff
@@ -109,7 +109,7 @@
 
 /obj/item/clothing/gloves/thick/swat/blackshield_corpsman
 	name = "corpsman combat gloves"
-	desc = "A pair of fire and impact-resistant corpsman combat gloves. This pair has an extra rubbery coating to help prevent germs."
+	desc = "A pair of fire and impact-resistant corpsman combat gloves." // No lying, it's not anti-germs.
```
Comment (Trilbyspaceclone) [inline code/modules/clothing/gloves/miscellaneous.dm:112]: it was to show it has extra bio armor

File: code/modules/mob/living/carbon/human/human_attackhand.dm [hunk 5]
```diff
@@ -179,9 +179,9 @@
 			if (H.stats.getStat(STAT_ROB) >= 80)
 				var softcap = H.stats.getStat(STAT_ROB) / 10
 				var newrob = (72 + softcap) / 10
-				stat_damage = 3 + max(0, newrob)
+				stat_damage = 3 + max(0, newrob) + H.punch_damage_increase // This makes knuckledusters actually work since NOTHING called this var, ever
```
Comment (Trilbyspaceclone) [inline code/modules/mob/living/carbon/human/human_attackhand.dm:182]:
```suggestion
				stat_damage = 3 + max(0, newrob)
```
Dont doble punch increases

File: code/modules/mob/living/carbon/human/human_attackhand.dm [hunk 6]
```diff
@@ -179,9 +179,9 @@
...
 			else
-				stat_damage = 3 + max(0, (H.stats.getStat(STAT_ROB) / 10))
+				stat_damage = 3 + max(0, (H.stats.getStat(STAT_ROB) / 10)) + H.punch_damage_increase
```
Comment (Trilbyspaceclone) [inline code/modules/mob/living/carbon/human/human_attackhand.dm:184]:
```suggestion
				stat_damage = 3 + max(0, (H.stats.getStat(STAT_ROB) / 10))
```

**Comment by Trilbyspaceclone** [comment]
Dont block off the upper area of the gardener room, thats used for blobs and hivemind

File: code/modules/mob/living/carbon/human/human_attackhand.dm (hunk 5 above)
Comment (ThePainkiller) [reply code/modules/mob/living/carbon/human/human_attackhand.dm:182]: I've tested this live, in many situations, with both dummies and roaches, this doesn't double punching damage, it caused the exact damage a normal punch at 40 ROB with knuckledusters SHOULD give
This var is called nowhere at all unless you put on the gloves that affect said var

**Comment by ThePainkiller** [comment]
> Dont block off the upper area of the gardener room, thats used for blobs and hivemind

With upper area you mean the part that's next to genetics?

File: code/modules/mob/living/carbon/human/human_attackhand.dm (hunk 6 above)
Comment (ThePainkiller) [reply code/modules/mob/living/carbon/human/human_attackhand.dm:184]: On a 40 ROB character (Foreman), it went from 8.9 damage with unarmed attacks to a naked human mob to 22 damage with gold knuckle dusters which provide +15 punching strength, (it should've been ~24)
Took off the gloves, punching damage is back to normal
Only the HULK mutation **doubles** punch increases, this is a `+` not a `*` meaning flat damage added

File: code/game/machinery/crafting_station.dm (hunk 3 above)
Comment (ThePainkiller) [reply code/game/machinery/crafting_station.dm:259]: Done!

File: code/game/machinery/crafting_station.dm (hunk 2 above)
Comment (ThePainkiller) [reply code/game/machinery/crafting_station.dm:252]: Oof thank you

File: code/modules/clothing/gloves/miscellaneous.dm (hunk 4 above)
Comment (ThePainkiller) [reply code/modules/clothing/gloves/miscellaneous.dm:112]: But it doesn't prevent germs and you didn't let me assign it the var Nitriles have >:|

**Comment by Trilbyspaceclone** [comment]
the one by the garden room

**Comment by Trilbyspaceclone** [comment]
image

**Comment by ThePainkiller** [comment]
> image

This is why I should use "replace", god damn it.

**Comment by DorfGuy1** [comment]
what does the jill haircut look like?

**Comment by ThePainkiller** [comment]
> what does the jill haircut look like?

The screenshot is right there at the top. It's a shorter version of the long twintails

**Pull request merged**